### ISSUE TITLE
Add shared resources

### ISSUE THREAD

**Issue by RaulFdzbis** [roboticslab-uc3m/xgnitive#20]
In the current implementation of the xgnitive project absolute paths are used in order to find the different resources needed for the system. 

As an improvement an implementation of a shared resources folder is proposed. Something similar to what is done in [teo-main](https://github.com/roboticslab-uc3m/teo-main/blob/develop/programs/teoSim/main.cpp#L67) . This way absoulute paths would be no longer needed, increasing the portability of the project.

**Comment by jgvictores** [comment]
Yes, models and configurations should go in https://github.com/roboticslab-uc3m/xgnitive/tree/develop/share
(maybe later on we will move models outside of the repository, but using the ResourceFinder is a great first step!)

**Comment by jgvictores** [comment]
I'm particularly concerned about:
https://github.com/roboticslab-uc3m/xgnitive/tree/develop/programs/models
How about a teo-openrave-models repository for the robot, and xgnitive scenarios within our own share ?

**Comment by RaulFdzbis** [comment]
Yes i like the idea, as far as i know the TEO openrave models folder is used at least in teo-main and xgnitive. So maybe could be a good idea to have that folder as an independent repository.

Just a question, what do you plan to add to that repository in addition to the models?. I think that maybe having something similar to a TeoSim standalone repository with the models could be nice.

**Comment by jgvictores** [comment]
My idea (related with https://github.com/roboticslab-uc3m/QA/issues/1 and https://github.com/roboticslab-uc3m/QA/issues/2) is:
- https://github.com/roboticslab-uc3m/openrave-yarp-plugins <- as is, with more capabilities (camera..)
- https://github.com/roboticslab-uc3m/teo-openrave-models <- new
- https://github.com/roboticslab-uc3m/teo-main <- lightweight, binds the previous two among others

**Comment by RaulFdzbis** [comment]
That's look fine to me. The only thing that still unclears is what would you put in the new teo-openrave-models repository.

**Comment by jgvictores** [comment]
Only the teo openrave models.

**Comment by RaulFdzbis** [comment]
Ok that can work, lets do it.

**Comment by RaulFdzbis** [comment]
Done! https://github.com/roboticslab-uc3m/teo-openrave-models

**Comment by jgvictores** [comment]
Cool,  looks good! Thanks!

**Comment by RaulFdzbis** [comment]
@jgvictores should we not keep this open until we configure xgnitive to work with the teo-openrave-models?

**Comment by jgvictores** [comment]
Okay!

**Comment by RaulFdzbis** [comment]
@jgvictores  Should do we close this with the last commit?

**Comment by jgvictores** [comment]
Still have to move things around, such as:
https://github.com/roboticslab-uc3m/xgnitive/tree/develop/programs/cgdaExecutionOET/conf
